package-lock.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@dydxprotocol/v4-localization",
-  "version": "1.1.47",
-  "lockfileVersion": 1,
+  "version": "1.1.48",
+  "lockfileVersion": 3,
```

Comment:
@rosepuppy   This is generated by new version of npm.

`ruihuang@Rh-dydx-mbp v4-localization % npm --version
10.2.4
`
Shouldn't matter, right?

Reply:
Yea should be good